### DISCUSSION TITLE
How to store HBA card information that are not being consumed within Netbox?

### DISCUSSION THREAD

**Discussion by sebastianesc** [netbox-community/netbox#17185] [Q&A]
Hi community, I have this question, in my inventory I have different HBA cards that are being continuously rotating between hosts. Looking through documentation the only space where I think I can fit this inventory is within Module Types and then to create Modules but I don't have it clear at all. 
As far as I understand is that a module type describes a model and it get's attached to the Module once you create one, but, what if I have more than 1 HBA of one single model? In my lab I have a name for each HBA, let's say HBA1234.. so what I want is to create my inventory for those HBAs and see if they are already being consumed by any device or if they are free.

So I created this:
![Captura de pantalla 2024-08-15 a la(s) 2 46 04 p m](https://github.com/user-attachments/assets/2d57f012-fc03-4c45-8d88-fbef10082a17)
Where I had to create two custom fields: Adapter Model (because Model Name field is being taken by the name HBA***) and Status.

As you can see, I have created two different HBA registers within the Module Type section and you may wonder.. why you didn't only create the register for the model? Because whenever I want to create a model it asks me for a device to attach that module, and that doesn't fit to my requirements. Not all my HBA are being consumed and even tho I need a space able to see that inventory about available HBAs.

Once I insert that HBA within my device, then I can properly create a Module register as now I have a proper device to assign in. What would I do with my remaining unused inventory?

So... is there a better approach to cover my requirement? I don't feel confortable with this approach and I want to know any other point of view. I also tried with the Inventory items which description totally fits with my HBA inventory but again, whenever I want to create an Inventory Item register it asks for a device where it will be inserted and I need space also for the once are not being consumed.

Hope I explained this clear and if not just let me know :) 
Thanks!

**Comment by a084ed22** [comment]
netbox is not an inventory management solution on its own. I'd advise to look at netbox-inventory if you want to extend it to work like one.
